calico-enterprise/threat/web-application-firewall.mdx

```diff
@@ -52,6 +52,17 @@ You simply enable WAF in Manager UI, and determine the services that you want to
 - WAF is not supported for host-networked client pods
 - When selecting and deselecting traffic for WAF, active connections may be disrupted
 
+:::caution
+
+Enabling WAF for certain system services may result in an undesired cluster state. Do not enable WAF for system service with the following prefixes:
+
+- `tigera-*`
+- `calico-*`
+- `kube-system`
+- `openshift-*`
```

Comment:
for openshift: should this be so? This has been updated so that it currently filters only the following
 {name: ‘openshift’, namespace: ‘default’}

Reply:
@BatoulSarvi Can you please confirm on this?

Reply:
Nevermind the previous comment it was based on a misunderstanding

Reply:
No worries.